src/admin/resources/js/cmsadmin.js

```diff
@@ -181,8 +186,12 @@
 				$scope.data.parent_nav_id = 0;
 				$scope.data.is_draft = 0;
 
-				$scope.data.nav_container_id = 1;
-
+				$scope.data.nav_container_id = ServiceCurrentWebsite.currentWebsite.default_container_id;
```

Comment:
This will throw an exception (and breaks the whole create page form) unless the object is resolved. 

If event `service:CurrentWebsiteChanged` is broadcasted anyway, then this line is not required. But i am not sure about that

The problem is that `$scope.data.nav_container_id = ServiceCurrentWebsite.currentWebsite.default_container_id;` will throw an type error `Cannot read property 'default_container_id' of undefined` because the state of service.currentWebsite object is by default 
`var service = {
		currentWebsite: null,`